templates/macros/filters/checkbox.html

```diff
@@ -33,3 +33,15 @@
   </fieldset>
 </div>
 {% endmacro %}
+
+{% macro checkbox_dropdown_multiple(name, label, selected={}, options={}) %}
```

Comment:
Yeah, I agree. The idea is that these are used when there's multiple dropdown components used in a single filter. Unless you have an idea for a name, I'm just going to add a comment.